### DISCUSSION TITLE
What advice do you have for getting off of nightly builds?

### DISCUSSION THREAD

**Discussion by sbauch** [parcel-bundler/parcel#5891] [Q&A]
I use parcel 2 nightly builds for my application. I can't really recall now, but I couldn't get the most recent beta to work with my app (I'd be happy to dig back in to that if helpful).

For a number of months this was fine. I set up dependabot and a github action for automerging dependency upgrades once CI passes. A little risky but again it's been fine, Parcel updates that break the build have never been merged, and there's only been one instance where I had to report an issue / make some changes for things to keep swimming along.

That all seemingly changed this week? It seems like nightly@600 started adding dependencies to my package.json when I build, and now I get a handful of parcel related nightly package bumps. Since dependabot is bumping one package per PR, they all fail on not being able to find the other packages at the required version.

See my list of [open PRs here](https://github.com/enterprise-oss/osso/pulls), where each fails in CI with errors like 

```
@parcel/package-manager: Could not find module "@parcel/transformer-css" satisfying 2.0.0-nightly.600+f28eaf60.
--
  | /var/lib/buildkite-agent/builds/enterprise-oss-bk-2/enterpriseoss/osso/package.json:27:5
  | 26 \|     "@parcel/packager-css": "2.0.0-nightly.600",
  | > 27 \|     "@parcel/transformer-css": "2.0.0-nightly.602",
  | >    \|     ^^^^^^^^^^^^^^^^^^^^^^^^^ Found this conflicting local requirement.
```

So I _think_ I could solve this by sticking a caret in front of all my parcel dependency versions in package.json? 

But I feel like the better solution is to stop living on the bleeding edge. Should I just pick a nightly build and stop using dependabot for parcel? Is a stable beta release planned soon?

I really love that I don't have to think about parcel much, but I'm a bit worried that I will now start each day with 6 busted parcel PRs and start to feel like I think about parcel too much! 

What would you do if you were me, other Parcel users and maintainers?

**Comment by DeMoorJasper** [comment]
I would stick it to a fixed nightly and manually update as the nightlies change quite often anyway. We're working towards the stable version and getting pretty close, so I think the next stable release will be a release candidate.

**Comment by basher** [comment]
For anyone else who stumbles on this, I've just found this really helpful article on [Parcel V2 Beta 3](https://v2.parceljs.org/blog/beta3/).

I'm currently running on `nightly 400` so I'm very keen to upgrade to V2! :)
